kits/sparktrace/constitutions/default.md

```
@@ -0,0 +1,102 @@
+# Default Constitution
+
+## Identity
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)